test/integration/util/util.go

```diff
@@ -764,7 +764,7 @@ func createNodes(cs clientset.Interface, prefix string, wrapper *st.NodeWrapper,
 	nodes := make([]*v1.Node, numNodes)
 	for i := 0; i < numNodes; i++ {
 		nodeName := fmt.Sprintf("%v-%d", prefix, i)
-		node, err := CreateNode(cs, wrapper.Name(nodeName).Obj())
+		node, err := CreateNode(cs, wrapper.Name(nodeName).Label("kubernetes.io/hostname", nodeName).Obj())
```

Comment:
cc @pohly This modified the util function, I think it's ok because this is a common label for node.

Reply:
Agreed.

Reply:
Can you help to tag a **Approve** as we need a test one? Thanks a lot.

Reply:
/approve